### DISCUSSION TITLE
Add a Wiki

### DISCUSSION THREAD

**Discussion by thebonbon** [EveronLife/EveronLife#65] [Ideas]
A Wiki would be a great place to explain systems or create tutorials.
Like how to create a new trader / crop type etc. Or how a system works in general.

If we document everything from the beginning it will be way easier for others to understand and expand upon systems.
_(Once a system is final and no breaking changes are made in the future)_

This could be done in Github or on a seperate website.

**Arma 3 ACE3** is a good example. Pretty sure they had their wiki on github before. Now its on a seperate website:
https://ace3.acemod.org/wiki/

**Some system examples:**
https://ace3.acemod.org/wiki/framework/fortify-framework.html
https://ace3.acemod.org/wiki/framework/medical-framework.html

**Comment by Arkensor** [comment]
Important tutorials that remain useful to server owners in the future like "how to add models" should all come with the `docs` inside this repo. A separate wiki somewhere else usually does not survive a maintainer change and is not included in forks.

**Reply by DrLuk** [reply]
Are the current docs made with the github wiki feature? I remember there being a function being built in.
